sdcm/cluster_k8s/__init__.py

```diff
@@ -403,17 +403,28 @@ def start_scylla_manager_journal_thread(self):
         self._scylla_manager_journal_thread = ScyllaManagerLogger(self, self.scylla_manager_log)
         self._scylla_manager_journal_thread.start()
 
-    def set_nodeselector_for_kubedns(self, pool_name):
-        # EKS and GKE deploy kube-dns pods, so make it be deployed on default nodes, not any other
+    def set_nodeselector_for_deployments(self, pool_name, namespace="kube-system", selector=""):
```

Comment:
Please, add type annotations for arguments and return value.

Reply:
Done